app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_clas: int, clean_mark: int, brand: str) -> None:
```

Comment:
The parameter name `comfort_clas` should be `comfort_class` to match the task description and ensure consistency.